src/services/NetworkService.ts

```diff
@@ -168,7 +168,7 @@ export class NetworkService {
     const repositoryFactory = new RepositoryFactoryHttp(url)
```

Comment:
How the wallet creates the RepositoryFactoryHttp is going to be simplified in the next SDK. The createLister won't need to be patched